web/src/lib/components/Profile.svelte

```diff
@@ -139,9 +139,7 @@
 <input type="checkbox" id="profile-modal-toggle" for="profile-modal" class="modal-toggle" />
 <div class="modal">
     <div class="modal-box relative flex justify-center items-center w-10/12 max-w-1xl">
-        {#if $user && !$user.hasAuctions}
-            <label for="profile-modal" class="btn btn-sm btn-circle absolute right-2 top-2" on:click={hide_or_logout}>✕</label>
-        {/if}
+        <label for="profile-modal" class:hidden={$user && $user.hasAuctions && $user.contributionPercent == null} class="btn btn-sm btn-circle absolute right-2 top-2" on:click={hide_or_logout}>✕</label>
```

Comment:
Please use `===` and `!==` for comparisons in Javascript, rather than `==` and `!=`. The old code used to be this way.